### ISSUE TITLE
[Feature Request]: Option to update to beta releases

### ISSUE THREAD

**Issue by theschles** [jordanbaird/Ice#210]
### Check existing feature requests

- [X] I have checked existing issues and believe that my feature request is not a duplicate

### Description

Hi! Thank you for putting into place #1 (notch macbook support)! Can't wait for when `0.10.0` is officially released.

Your users are interested in trying out beta releases.  I suggest adding another toggle swtich in the "Updates" section for "include beta releases".  

That way -- with the options that are currently there for "Automatically check for updates" and "Automatically download updates" -- and if users are interested -- users will additionally have the option to be, respectively:
* informed when a beta release is available for download
* have that beta release automatically downloaded to their computer

Please keep up the good work!

**Comment by todddube** [comment]
Good Morning - might be similar love the app truly! Left BarTender for your app - I'm on .10 beta 5 but no update showing for beta 6? It seems to be looking at the main release?  screen shot attached 
![Uploading SCR-20240708-hfcb.png…]()

**Comment by jordanbaird** [comment]
@todddube Yeah, currently auto updates only work for main releases. I need to get some automation in place before I start enabling them for beta releases.

**Comment by Alexandros-Kordatzakis** [comment]
Reminder for this feature! Would like to be able to auto update, from the app and the terminal, for beta updates! 

Thank you for the lovely app!
